docs/get-started/getting-started-databricks.md

```diff
@@ -68,41 +68,15 @@ cluster meets the prerequisites above by configuring it as follows:
 3. Select a worker type. On AWS, use nodes with 1 GPU each such as `p3.2xlarge` or `g4dn.xlarge`.
    For Azure, choose GPU nodes such as Standard_NC6s_v3. For GCP, choose N1 or A2 instance types with GPUs. 
 4. Select the driver type. Generally this can be set to be the same as the worker.
-5. Start the cluster.
-
-## Advanced Cluster Configuration
-
-We will need to create an initialization script for the cluster that installs the RAPIDS jars to the
-cluster.
-
-1. To create the initialization script, import the initialization script notebook from the repo to
-   your workspace.  See [Managing
-   Notebooks](https://docs.databricks.com/notebooks/notebooks-manage.html#id2) for instructions on
-   how to import a notebook.  
-   Select the version of the RAPIDS Accelerator for Apache Spark based on the Databricks runtime
-   version:
-   - [Databricks 10.4 LTS
-     ML](https://docs.databricks.com/release-notes/runtime/10.4ml.html#system-environment) has CUDA 11
-     installed.  Users will need to use 22.04.0 or later on Databricks 10.4 LTS ML.
-   - [Databricks 11.3 LTS
-     ML](https://docs.databricks.com/release-notes/runtime/11.3ml.html#system-environment) has CUDA 11
-     installed.  Users will need to use 23.02.0 or later on Databricks 11.3 LTS ML.
-     
-     In both cases use
-     [generate-init-script.ipynb](../demo/Databricks/generate-init-script.ipynb) which will install
-     the RAPIDS Spark plugin.
-
-2. Once you are in the notebook, click the “Run All” button.
-3. Ensure that the newly created init.sh script is present in the output from cell 2 and that the
-   contents of the script are correct.
-4. Go back and edit your cluster to configure it to use the init script. To do this, click the
-   “Compute” button on the left panel, then select your cluster.
-5. Click the “Edit” button, then navigate down to the “Advanced Options” section. Select the “Init
-   Scripts” tab in the advanced options section, and paste the initialization script:
-   `dbfs:/databricks/init_scripts/init.sh`, then click “Add”.
-
-    ![Init Script](../img/Databricks/initscript.png)
-
+5. We will need to create an initialization script for the cluster that installs the RAPIDS jars to the
+   cluster. Databricks recommends storing all cluster-scoped init scripts using workspace files. 
+   Each user has a Home directory configured under the /Users directory in the workspace. 
+   Navigate to your home directory in the UI and select **Create** > **File** from the menu, 
+   create an `init.sh` scripts with contents:   
+   ```bash
+   #!/bin/bash
+   sudo wget -O /databricks/jars/rapids-4-spark_2.12-23.04.0.jar https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.04.0/rapids-4-spark_2.12-23.04.0.jar
+   ```
```

Comment:
This makes the user interrupt their flow of Cluster creation. We are telling them to click away from the cluster config edit. Can we move it up as a preparation step?

Here we should only talk about specifying the init script created in the preparation step. 

> Add the init script path from Step X , e.g., `/Users/user@domain/init.sh`, with the default destination type `Workspace`.

Reply:
move it to preparation step, thanks